### PR TITLE
update python-client docs

### PR DESCRIPTION
**Pull request by StefanBogdan**

### CODE REVIEW

File: developers/weaviate/current/client-libraries/python.md [hunk 1]
```diff
@@ -524,8 +554,9 @@ The `Batch` object can be configured using the `batch.configure()` method or the
...
 - `callback` (Optional[Callable[[dict], None]]: default `weaviate.util.check_batch_result`): It is a callback function on the results of the `batch.create_objects()` and `batch.create_references()`. It is used for Error Handling for Auto-/Dynamic-batching. Has no effect if `batch_size` is `None`.
-- `timeout_retries` - (`int`: default `0`): Number of times to re-try to import/create a batch that resulted in `TimeoutError`.
-
+- `timeout_retries` - (`int`: default `3`): Number of re-try to do on import/create a batch that resulted in `TimeoutError`.
+- `connection_error_retries` - (`int`: default `3`): Number of re-try to do on import/create a batch that resulted in `ConnectionError`. **NOTE:** Available in `weaviate-client>=3.9.0`.
+- `num_workers` - (`int`: default `0`): The maximal number of concurrent threads to run batch import. Only used for non-MANUAL batching. i.e. is used only with AUTO or DYNAMIC batching. ***Use with care to not overload your weaviate instance.*** **NOTE:** Available in `weaviate-client>=3.9.0`.
```
Comment (dirkkul) [inline developers/weaviate/current/client-libraries/python.md:559]: default is 1

File: developers/weaviate/current/client-libraries/python.md [hunk 2]
```diff
@@ -158,6 +173,21 @@ Batching is a way of importing/creating `objects` and `references` in bulk using
 2. ***Dynamic-batching***
 3. ***Manual-batching***
 
+## New: Multi-threading batch import (weaviate-client>=3.9.0)
+Python client version `3.9.0` introduces Multi-threading Batch import which works with both `Auto-batching` and `Dynamic-batching`. You can set the number of workers (threads) can be configured using the `.configure(...)` (same as `.__call__(...)`) by setting the argument `num_workers`. See also *Batch-configuration* below.
```
Comment (dirkkul) [inline developers/weaviate/current/client-libraries/python.md:177]: I would add that this is disabled by default

**Review by dirkkul — APPROVED** [review]

File: developers/weaviate/current/client-libraries/python.md (hunk 1 above)
Comment (StefanBogdan) [reply developers/weaviate/current/client-libraries/python.md:559]: Oooo, my bad. It cannot actually be set to 0 )))

File: developers/weaviate/current/client-libraries/python.md (hunk 2 above)
Comment (StefanBogdan) [reply developers/weaviate/current/client-libraries/python.md:177]: Agree

**Review by dirkkul — APPROVED** [review]

File: developers/weaviate/current/client-libraries/python.md [hunk 3]
```diff
@@ -46,9 +47,23 @@ secret = weaviate.AuthClientCredentials("secret")
...
+import weaviate
+
+# Initiate the client with the secret
+client = weaviate.Client(
+  "https://localhost:8080",
+  additional_headers={"authorization": "Bearer <MY_TOKEN>"},
+)
+```
+
+If you want to authenticate to Weaviate using a bearer toke
```
[the remark this answers is not in the document]
Comment (StefanBogdan) [reply developers/weaviate/current/client-libraries/python.md:65]: Sure

[the remark this answers is not in the document]
Comment (StefanBogdan) [reply developers/weaviate/current/client-libraries/python.md]: Forgot to delete it.

**Pull request merged**